lxd/device/nic_bridged.go

```diff
@@ -575,7 +581,7 @@ func (d *nicBridged) setFilters() (err error) {
 	if shared.IsTrue(d.config["security.ipv6_filtering"]) {
 		// Check br_netfilter is loaded and enabled for IPv6.
 		sysctlPath := "bridge/bridge-nf-call-ip6tables"
-		sysctlVal, err := NetworkSysctlGet(sysctlPath)
+		sysctlVal, err := util.SysctlGet("/net", sysctlPath)
```

Comment:
@keyallis please can this just be `sysctlVal, err := util.SysctlGet(sysctlPath)` with `sysctlPath := "net/bridge/bridge-nf-call-ip6tables"`